helm: install rook in openshift cluster (#12675)
rook/rook: Documentation/Getting-Started/ceph-openshift.md

oc create -f cluster.yaml
```

## Helm Installation

> Use the [Helm Documentation](../Helm-Charts/helm-charts.md]. Openshift related configuration is auto-discver.

Reviewer: ```suggestion
Configuration required for Openshift is automatically created by the Helm charts, such as the SecurityContextConstraints. See the [Rook Helm Charts](../Helm-Charts/helm-charts.md].
```

oc create -f cluster.yaml
```

## Helm Installation

> Set the `openshift` to `true` in `rook-release/rook-ceph` values.yaml.

Author: Used the second approach

> Use the [Helm Documentation](/Documentation/Helm-Charts/helm-charts.md) for further steps.

Author: This one is better

oc create -f cluster.yaml
```

> Helm Installation:

Reviewer: ```suggestion
## Helm Installation
```

> Using helm Installation:

Reviewer 2: ```suggestion
Helm Installation:
```

> Set the `openshift` to `true` in `rook-release/rook-ceph` values.yaml.

Reviewer: Let's just mention this setting, and then link to the helm chart doc (operator-chart.md). No need to put the same helm commands here too.

oc create -f cluster.yaml
```

## Helm Installation

> Configuration required for Openshift is automatically created by the Helm charts, such as the SecurityContextConstraints. See the [Rook Helm Charts](../Helm-Charts/helm-charts.md].

Reviewer: The closing parenthesis is incorrect, and need a newline before the next section.
```suggestion
Configuration required for Openshift is automatically created by the Helm charts, such as the SecurityContextConstraints. See the [Rook Helm Charts](../Helm-Charts/helm-charts.md).

```
